src/views/BackgroundInfoView.vue

```diff
@@ -98,34 +97,150 @@
 
           <li class="uc-form-col">
             <p>
-              How much prior tutoring and/or college counseling experience do
-              you have?<span class="background-info__question-required">*</span>
-            </p>
-            <p v-if="showInputErrors && !experience" class="error">
-              Please fill out this field.
+              Where do you currently live?<span
+                class="background-info__question-required"
+                >*</span
+              >
             </p>
 
-            <div
-              v-for="option in options.experience"
-              :key="option"
-              class="uc-form-radio"
+            <label class="uc-form-label location-label">Country</label>
+            <select
```

Comment:
We have a component called `v-select` that might improve usability here (pretty sure it supports searching for options)